lms/djangoapps/branding/api.py

```diff
@@ -461,25 +479,22 @@ def _footer_logo_img(is_secure):
     # EdX needs the FOOTER_ORGANIZATION_IMAGE value to point to edX's
     # logo by default, so that it can display properly on edx.org -- both
     # within the LMS, and on the Drupal marketing site, which uses this API.
-    try:
-        return _absolute_url_staticfile(is_secure, logo_name)
-    except ValueError:
-        # However, if the edx.org comprehensive theme is not activated,
-        # Django's staticfiles system will be unable to find this footer,
-        # and will throw a ValueError. Since the edx.org comprehensive theme
-        # is not activated by default, we will end up entering this block
-        # of code on new Open edX installations, and on sandbox installations.
-        # We can log when this happens:
-        default_logo = "images/logo.png"
-        log.info(
-            u"Failed to find footer logo at '%s', using '%s' instead",
-            logo_name,
-            default_logo,
-        )
-        # And we'll use the default logo path of "images/logo.png" instead.
-        # There is a core asset that corresponds to this logo, so this should
-        # always succeed.
-        return staticfiles_storage.url(default_logo)
+    if footer_url_from_site_config:
+        return _absolute_url_staticfile(is_secure, footer_url_from_site_config)
+
+    if brand_footer_logo_url:
+        return brand_footer_logo_url
+
+    log.info(
+        "Failed to find footer logo at '%s', using '%s' instead",
+        footer_url_from_site_config,
+        default_local_path,
+    )
```

Comment:
The log should mention the footer from both site config & settings is missing/None

Reply:
+1 on that.